gobblin-utility/src/main/java/org/apache/gobblin/util/ConfigUtils.java

```diff
@@ -561,7 +562,6 @@ public static Config resolveEncrypted(Config config, Optional<String> encConfigP
     Config encryptedConfig = config.getConfig(encConfigPath.get());
 
     PasswordManager passwordManager = PasswordManager.getInstance(configToProperties(config));
-    Map<String, String> tmpMap = Maps.newHashMap();
```

Comment:
deleted by mistake?

Reply:
Oops yeah, that was supposed to be in the next PR but I missed the glob, will fix